sidebars.js

```diff
@@ -14,140 +14,222 @@ module.exports = {
+              label: 'Contribution guidelines',
+              href: 'https://github.com/hirosystems/clarinet#contributing',
             },
+            
             {
               type: 'link',
-              label: 'Types',
-              href: 'https://docs.stacks.co/docs/write-smart-contracts/clarity-language/language-types',
+              label: 'Clarinet - Discord',
+              href: 'https://discord.com/channels/621759717756370964/839633619261456444',
             },
             {
               type: 'link',
-              label: 'Keywords',
-              href: 'https://docs.stacks.co/docs/write-smart-contracts/clarity-language/language-keywords',
+              label: 'Blogs',
+              href: 'https://www.hiro.so/search?query=Clarinet',
             },
             {
               type: 'link',
-              label: 'Functions',
-              href: 'https://docs.stacks.co/references/language-functions',
+              label: 'Videos',
+              href: 'https://www.youtube.com/c/HiroSystems/search?query=Clarinet',
             },
           ],
         },
+      ],
+    },
+    {
+      type: 'category',
+      label: 'Stacks.js',
+      items: [
+        
+        'stacks.js/overview',
+        'stacks.js/getting-started',
         {
           type: 'category',
-          label: 'Stacks.js',
+          label: 'Feature Guides',
+          items: [
+            'stacks.js/feature-guides/authenticate-users-with-connect',
+            'stacks.js/feature-guides/sign-messages',
+            'stacks.js/feature-guides/sign-transactions',
+            'stacks.js/feature-guides/store-data-securely',
+          ],
+        },
+        {
+          type: 'category',
+          label: 'How-to Guides',
+          items: [
+            'stacks.js/how-to-guides/how-to-integrate-stacking-delegation',
+            'stacks.js/how-to-guides/how-to-integrate-stacking',
+            'stacks.js/how-to-guides/how-to-migrate-from-blockstack.js',
+            'stacks.js/how-to-guides/how-to-use-stacks-connect-with-angular',
+          ],
+        },
+        'stacks.js/faq',
+        {
+          type: 'category',
+          label: 'Developer Resources',
           items: [
             {
               type: 'link',
-              label: 'blockchain-api-client',
-              href: 'https://hirosystems.github.io/stacks-blockchain-api/client/index.html',
-            },
-            {
-              type: 'link',
-              label: 'connect',
-              href: 'https://github.com/hirosystems/connect#readme',
-            },
-            {
-              type: 'link',
-              label: 'auth',
-              href: 'https://stacks.js.org/modules/auth.html',
-            },
-            {
-              type: 'link',
-              label: 'storage',
-              href: 'https://stacks.js.org/modules/storage.html',
+              label: 'Contribution guidelines',
```

Comment:
```suggestion
              label: 'Contribution Guidelines',
```

Let's use this case everywhere